bug-1668861-pref-measure-set-to-default-adoption-impact-of-chang-release-81-83.toml

```diff
@@ -0,0 +1,27 @@
+[experiment]
+reference_branch = "modal-default-browser-notification"
+
+[metrics]
+daily = ["was_ever_default_browser", "was_never_default_browser"]
+weekly = ["was_ever_default_browser", "was_never_default_browser", "retained", "active_hours", "uri_count", "search_count"]
+overall = ["was_ever_default_browser", "was_never_default_browser", "active_hours", "uri_count", "search_count"]
+
+[metrics.was_ever_default_browser]
+select_expression = "LOGICAL_OR(is_default_browser)"
+data_source = "clients_daily"
+
+[metrics.was_ever_default_browser.statistics.binomial]
+pre_treatments = ["remove_nulls"]
+
+[metrics.was_never_default_browser]
+select_expression = "LOGICAL_AND(NOT is_default_browser)"
```

Comment:
@tdsmith Note that this is slightly different from the one defined [here](https://github.com/mozilla/jetstream-config/blob/main/bug-1665467-pref-shirley-omnibus-experiment-release-81-82.toml#L16). Does this make sense to you?

Reply:
Ah, you're right, that was "was ever not default" and this one is "was never default". Good catch!